react/components/ProductSummaryPrice/ProductSummaryPrice.js

```diff
@@ -101,6 +103,8 @@ ProductSummaryPrice.propTypes = {
   showLabels: PropTypes.bool,
   /** Set installments' visibility */
   showInstallments: PropTypes.bool,
+  /** Set savings' visibility*/
+  showSavings: PropTypes.bool,
```

Comment:
Maybe this could be an `enum`, to be more extensible in the future if, for example, we add the functionality of showing the discount percentage